packages/odyssey-storybook/src/components/Tag/Tag.stories.tsx

```diff
@@ -27,9 +27,25 @@ export default {
   },
 };
 
-const Template: Story<TagProps> = ({ tags }) => <Tag tags={tags} />;
+const Template: Story<TagProps> = ({ children }) => <Tag>{children}</Tag>;
 
 export const Default = Template.bind({});
 Default.args = {
-  tags: ["Item one", "Item two", "Item three"],
+  children: "Item",
 };
+
+export const Multiple = Template.bind({});
+Multiple.args = {
+  children: "Item",
+};
+Multiple.decorators = [
+  (Story) => {
+    return (
+      <>
+        <Story />
+        <Story />
+        <Story />
+      </>
+    );
+  },
+];
```

Comment:
Overkill to have a different story for "multiple" tags? Should i add the decorator to `default` instead?